### ISSUE TITLE
Running Test failure : Cannot find an overload for "ToString" and the argument count: "2"

### ISSUE THREAD

**Issue by YvesDesrumaux** [microsoft/navcontainerhelper#1536]
**Describe the issue**
BcContainerHelper version 1.0.17-preview283, while using Run-AlPipeline. The running test section failed.

**Output of scripts**
```
  _____                   _               _______       _       
 |  __ \                 (_)             |__   __|     | |      
 | |__) |   _ _ __  _ __  _ _ __   __ _     | | ___ ___| |_ ___ 
 |  _  / | | | '_ \| '_ \| | '_ \ / _` |    | |/ _ \ __| __/ __|
 | | \ \ |_| | | | | | | | | | | | (_| |    | |  __\__ \ |_\__ \
 |_|  \_\__,_|_| |_|_| |_|_|_| |_|\__, |    |_|\___|___/\__|___/
                                   __/ |                        
                                  |___/                         

at Run-Tests, C:\ProgramData\BcContainerHelper\Extensions\isatech-fbie\PsTestTool\PsTestFunctions.ps1: line 435
at <ScriptBlock>, <No file>: line 57
Cannot find an overload for "ToString" and the argument count: "2".
at Run-Tests, C:\ProgramData\BcContainerHelper\Extensions\isatech-fbie\PsTestTool\PsTestFunctions.ps1: line 435
at <ScriptBlock>, <No file>: line 57
```

**Additional context**
- *It not always happens*: While running exactly the same Pipeline, it happens sometimes directly on first test running, sometimes on testing the one of the additionals countries, somtimes never...
- It use to work on version 1.0.15.

**Comment by YvesDesrumaux** [comment]
Today on 3 runs of the **same** pipeline, it fails on:
- First additional country (after 30min)
- Main country (after 10min)
- 15th additional country (after 3H20min)

I do not manage to understand why.
I tri again crossing fingers

**Comment by freddydk** [comment]
I will add some resilience to this piece.

**Comment by freddydk** [comment]
The latest preview of BcContainerHelper should have some resilience in this area and shouldn't fail, but instead write N/A if it cannot calculate a value.
If possible, please send me a test output file here or on email (freddyk at microsoft dot com) - possibly together with the full pipeline output. I would like to understand in which case this could/can go wrong.

**Comment by freddydk** [comment]
Shipped in 1.0.17
